addon/services/ajax.js

```diff
@@ -113,9 +113,9 @@ export default Ember.Service.extend({
         );
 
         if (response instanceof AjaxError) {
-          reject(response);
+          Ember.run.join(null, reject, response);
```

Comment:
I referring to this https://github.com/switchfly/ember-test-helpers/blob/master/lib/ember-test-helpers/wait.js#L37